### DISCUSSION TITLE
Display music genre tags on artist pages and use them to hide/show releases

### DISCUSSION THREAD

**Discussion by DameGAD** [navidrome/navidrome#2498] [Ideas]
Currently genre tags are displayed on release pages but not on artist pages. The idea is to display genre tags on the artist page, and be able to click these tags to hide/show releases tagged with a specific genre. Example of use cases:
- artists who recorded many different music genres. Cher is a good exemple.
- any artist singing in multiple languages for different markets providing that their albums are tagged with language related genre tags. Korean artists having both a career in South Korea and in Japan are a good example. It would allow to easily only show their Korean discrography or their Japanese discography. Celion Dion is another good example.
There are probably other possible use cases, but these are the ones I can think of.

**Comment by certuna** [comment]
Yes, this is a good idea - I've been thinking about writing a quick filter on Genre and Min/Max Year into the albums, songs and artist views, it's prefectly doable but this probably has to wait for the UI migration to ReactAdmin v4.
